src/TriboroughBridge-ChorusPlugin/CommandLineProcessor.cs

```diff
@@ -84,7 +84,7 @@ public static class CommandLineProcessor
 							//This is an unparsable command line
 							Console.WriteLine(CommonResources.kInvalidCommandLineOptions);
 							//Signal FLEx or other apps
-							throw new ApplicationException(String.Format("Invalid command line options: {0}.", args));
+							throw new ApplicationException(String.Format("Invalid command line options: {0}. Please launch from FLEx.", args));
```

Comment:
Unless there are objections, I'll leave this message intact, since it handles a different case.

Incidentally, this block could use more work (Console.WriteLine doesn't get a chance to do anything before we throw, and `args` shows up as 'String[]' in the exception message), but I'm not sure if it would be worth the effort to fix that.